css/templates/travel/components/_footer.css

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ .travel-footer {
+  background-color: var(--white);
```

Comment:
prefix all color vars with --color-

Reply:
We have not used this convention in merging in the other two PRs

Reply:
i do remember recommending it on your other PR , that needs to be fixed too

Reply:
Ok, I'll replace